atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepableCells.java

```diff
@@ -281,6 +287,11 @@ private long getTimestamp(SweepableCellsTable.SweepableCellsRow row, SweepableCe
         return row.getTimestampPartition() * SweepQueueUtils.TS_FINE_GRANULARITY + col.getTimestampModulus();
     }
 
+    private boolean knownToBeCommittedAfterSweepTs(long startTs, long sweepTs) {
+        Optional<Long> commitTsIfCached = commitTsCache.loadIfCached(startTs);
+        return commitTsIfCached.isPresent() && commitTsIfCached.get() >= sweepTs;
```

Comment:
nit: this is equivalent to `commitTsIfCached.map(x -> x >= sweepTs).orElse(false)` I think (and you can make the whole thing fluent if you want from there!)